src/lib/logical_query_plan/stored_table_node.cpp

```diff
@@ -43,16 +47,10 @@ const std::vector<ColumnID>& StoredTableNode::pruned_column_ids() const { return
 
 std::string StoredTableNode::description() const {
   std::ostringstream stream;
-  stream << "[StoredTable] Name: '" << table_name << "'";
-
-  if (!_pruned_chunk_ids.empty()) {
-    stream << " pruned chunks: " << _pruned_chunk_ids.size();
-  }
-
-  if (!_pruned_column_ids.empty()) {
-    stream << " pruned columns: " << _pruned_column_ids.size() << "/"
-           << StorageManager::get().get_table(table_name)->column_count();
-  }
+  stream << "[StoredTable] Name: '" << table_name << "' pruned: ";
+  stream << _pruned_chunk_ids.size() << " chunk(s), ";
```

Comment:
Any particular reason why showing the initial column count (which I like) but not the initial chunk count?

Reply:
Chunk counts are not fixed, column counts are relatively fixed, that's the only reason. Not sure if it's a good one

Reply:
With fixed you mean that the chunk count could be outdated when the information is printed? That would be always the case, right?

Reply:
> With fixed you mean that the chunk count could be outdated when the information is printed?

Yes pretty much... But as I said that's probably not a good argument. I'll change it to include "unpruned chunk count"

Reply:
updated OP with pictures of this